template-only-test/template_infra_test.go

```diff
@@ -177,7 +177,7 @@ func ValidateDevEnvironment(t *testing.T) {
 		TerraformDir: "../infra/app/envs/dev/",
 	})
 	serviceEndpoint := terraform.Output(t, terraformOptions, "service_endpoint")
-	http_helper.HttpGetWithRetryWithCustomValidation(t, serviceEndpoint, nil, 5, 1*time.Second, func(responseStatus int, responseBody string) bool {
+	http_helper.HttpGetWithRetryWithCustomValidation(t, serviceEndpoint, nil, 10, 3*time.Second, func(responseStatus int, responseBody string) bool {
```

Comment:
Maybe we can revert this change now that we found the actual cause?

Reply:
Yeap, I had something come up at the end of the day, so I wasn't able to see the last run was successful until this morning. Will be reverted in the next push

Reply:
I had to bump the retries amount back up, the service wasn't becoming accessible in the specified retry config. Bumping it back up to 10 fixed it timing out, so I think we should probably keep it at 10

Reply:
oh ok, sounds good, thanks!